### DISCUSSION TITLE
Does go agent 0.1.0 to log work well and what I should most carefully?

### DISCUSSION THREAD

**Discussion by xiaozaiyuji** [apache/skywalking#11208] [Q&A]
I did a test for go agent 0.1.0 to zap and Logrus with a native httpserver。but I did not print the 
![image](https://github.com/apache/skywalking/assets/4230320/6c8840fb-9a1a-41e4-a734-6a06a0472222)
trace info ,did I config this wrong? I check the go version and zap logrus version,but this is in the support list.So I was confusion.I test this with a very simple code. Can any one give me some advice.

**Comment by wu-sheng** [comment]
0.2 is out. You could try.

But, basically, you should check step by step. Is your OAP server, UI and database working? Does agent send the data?

Don't try to run the agent without backend.

**Reply by xiaozaiyuji** [reply]
Every thing is ok but log trace info.I was so confused.

**Reply by xiaozaiyuji** [reply]
This is the logrus test.
![image](https://github.com/apache/skywalking/assets/4230320/1a23f013-530c-4939-a5e5-3cadc3390520)
![image](https://github.com/apache/skywalking/assets/4230320/6ec732f6-a533-4c84-b67b-a8e2fc01abde)
This is the zap test.
![image](https://github.com/apache/skywalking/assets/4230320/3bac4ef6-8d3c-4672-9927-c3ac3a6e04c2)
![image](https://github.com/apache/skywalking/assets/4230320/a44cd7aa-6b46-40d2-85d8-67d1396b77fa)
I did the test so simple,but the log is not effective.

**Reply by mrproliu** [reply]
Please follow the [official documentation](https://skywalking.apache.org/docs/skywalking-go/next/en/setup/gobuild/) to add the `skywalking-go` package to your project, and import into the `main` package.

**Reply by xiaozaiyuji** [reply]
I did this as official documentation.But to my surprise the trace the metric show well except the log.

**Reply by xiaozaiyuji** [reply]
I had already do it.
![image](https://github.com/apache/skywalking/assets/4230320/fc03d6ec-6f57-4741-8e87-5f23c6b1e830)
